clueless-client/src/main/java/edu/jhu/espresso/client/ClueLessClient.java

```diff
@@ -0,0 +1,86 @@
+package edu.jhu.espresso.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+public class ClueLessClient implements Runnable
+{
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    private final String host;
+    private final int port;
+    private final Socket socket;
+    private final PrintWriter printWriter;
+    private final BufferedReader input;
```

Comment:
I like to have immutable objects, so I'll strive for `private final` modifiers on instance fields